internal/terraform/aws/dyanmodb_table.go

```diff
@@ -0,0 +1,114 @@
+package aws
```

Comment:
There's a typo in the filename here `dyanmodb_table.go` -> `dynamodb_table.go`

Reply:
Fixed